### PR TITLE
(sonarlint-vs2015) Migrate to core team packages

### PR DESCRIPTION
**Pull request by pascalberger**
Migrate from [my own repository](https://github.com/pascalberger/chocolatey-packages). Also changed to embedded package.

Chocolatey user is invited.

PR for removing from original repository: https://github.com/pascalberger/chocolatey-packages/pull/113. I will remove from my repo as soon as this is ready to be merged.

### CODE REVIEW

File: automatic/vs-extensions/sonarlint-vs2015/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿$ErrorActionPreference = 'Stop';
+
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+$filePath = "$toolsPath\SonarLint.VSIX-2.9.0.355-2015.vsix"
+
+$vsixUrl =  "file://" + $filePath.Replace("\", "/")
```
Comment (AdmiringWorm) [inline automatic/vs-extensions/sonarlint-vs2015/tools/chocolateyInstall.ps1]: If I remember correctly, I believe @ferventcoder said that although this works, it isn't really supported.

File: automatic/vs-extensions/sonarlint-vs2015/sonarlint-vs2015.nuspec [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+SonarLint is a Visual Studio 2015 extension that provides on-the-fly feedback to developers on new bugs and quality issues injected into C# and VB.NET code. SonarLint for Visual Studio is based on and benefits from the .NET Compiler Platform ("Roslyn") and its code analysis API to provide a fully-integrated user experience in Visual Studio 2015. With version 2.0 you can now establish a connection to a SonarQube server and bind your Visual Studio solution to a SonarQube project. This operation automatically updates the rulesets of the solution and attaches the required Roslyn analyzers to the solution.
+    </description>
+    <tags>sonarlint sonarqube sonarsource visualstudio extension foss</tags>
+    <releaseNotes>http://www.sonarlint.org/visualstudio/#News</releaseNotes>
+  </metadata>
+  <files>
+    <file src="legal\**" target="legal" />
+    <file src="tools\**" target="tools" />
+  </files>  
+</package>
```
Comment (AdmiringWorm) [inline automatic/vs-extensions/sonarlint-vs2015/sonarlint-vs2015.nuspec]: should be an empty line at the end of the file

File: automatic/vs-extensions/sonarlint-vs2015/update.ps1 [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  $version = $json.version | Select-Object -first 1
+
+  @{
+    Version   = $version
+    URL32     = $url
+    Filename32  = $filename
+  }
+}
+
+update -ChecksumFor none
```
Comment (AdmiringWorm) [inline automatic/vs-extensions/sonarlint-vs2015/update.ps1]: should be an empty line at the end of the file

**Comment by AdmiringWorm** [comment]
other than the last two comments, this looks good to me.

File: automatic/vs-extensions/sonarlint-vs2015/sonarlint-vs2015.nuspec (hunk 2 above)
Comment (pascalberger) [inline automatic/vs-extensions/sonarlint-vs2015/sonarlint-vs2015.nuspec]: Added. And fixed also encoding of the file

File: automatic/vs-extensions/sonarlint-vs2015/update.ps1 (hunk 3 above)
Comment (pascalberger) [inline automatic/vs-extensions/sonarlint-vs2015/update.ps1]: Added. And fixed also encoding of the file

**Comment by pascalberger** [comment]
@gep13 Can you please accept the invitation for the Chocolatey user?

**Comment by gep13** [comment]
>@pascalberger said...
Can you please accept the invitation for the Chocolatey user?

Done.

**Comment by pascalberger** [comment]
I propose to not push it at the moment and wait for the next update, since it already has a 4-digit version number

**Comment by gep13** [comment]
Sounds good to me :+1:

**Pull request merged**